### PR TITLE
docs: enable AWS STS

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR provides instructions on how to enable AWS STS for tenants. 

## Changed Pages

💻 [Enable Adding AWS Accounts using STS](https://deploy-preview-5101--docs-spectrocloud.netlify.app/enterprise-version/system-management/configure-aws-sts-account/)

## Jira Tickets

🎫 [DOCS-1515](https://spectrocloud.atlassian.net/browse/DOC-1515)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

### CODE REVIEW

File: _partials/self-hosted/_aws-sts-config.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+2. Log in to the {props.edition} System API by using the `/v1/auth/syslogin` endpoint. 
+   Use the curl command below and replace the `example.com` placeholder URL with the URL of your {props.edition} instance. 
+   Ensure you replace the credentials below with your system console credentials.
+
+   ```shell
+   TOKEN=$(curl --insecure --location 'https://example.com/v1/auth/syslogin' \
+    --header 'Content-Type: application/json' \
+    --data '{
+    "password": "**********",
+    "username": "**********"
```
Comment (addetz) [inline _partials/self-hosted/_aws-sts-config.mdx:37]: Indentation is off

File: _partials/self-hosted/_aws-sts-config.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+3. Next, prepare a payload for the AWS account you want to configure. 
+   Use the following JSON payload as a template and replace the `accessKey`, `secretKey`, and `accountId` fields with the AWS access key, secret key, and account ID of your AWS account.
+
+    ```json
+    CONFIG_JSON=$(cat <<EOF
+    {
+    "accessKey": "**********",
+    "secretKey": "**********",
+    "accountId": "123456789"
```
Comment (addetz) [inline _partials/self-hosted/_aws-sts-config.mdx:49]: Indentation is off

File: docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-dynamic.md [hunk 3]
```diff
@@ -46,6 +44,11 @@ cloud account.
 
 ### Prerequisites
 
+- If you are using a self-hosted Palette or Vertex instance, you have configured an AWS account at the instance-level to
```
Comment (addetz) [inline docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-dynamic.md:47]:
```suggestion
- If you are using a self-hosted Palette or Vertex instance, you must configure an AWS account at the instance-level to
```

File: docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-dynamic.md [hunk 4]
```diff
@@ -220,10 +219,13 @@ A multi-cloud account scenario requires you to perform the following authenticat
 Use the following steps to add an S3 bucket as the backup location using the STS authentication method when you have
 multiple cloud accounts.
 
-<br />
-
 ### Prerequisites
 
+- If you are using a self-hosted Palette or Vertex instance, you have configured an AWS account at the instance-level to
```
Comment (addetz) [inline docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-dynamic.md:224]:
```suggestion
- If you are using a self-hosted Palette or Vertex instance, you must configure an AWS account at the instance-level to
```

File: docs/docs-content/clusters/public-cloud/aws/add-aws-accounts.md [hunk 5]
```diff
@@ -49,6 +49,10 @@ Use the steps below to add an AWS cloud account using Security Token Service (ST
 
 #### Prerequisites
 
+- If you are using a self-hosted instance of Palette or VerteX, you have configured an AWS account at the instance-level
```
Comment (addetz) [inline docs/docs-content/clusters/public-cloud/aws/add-aws-accounts.md:52]:
```suggestion
- If you are using a self-hosted instance of Palette or VerteX, you must configure an AWS account at the instance-level
```

File: docs/docs-content/clusters/public-cloud/aws/add-aws-accounts.md [hunk 6]
```diff
@@ -161,6 +165,10 @@ Use the steps below to add an AWS cloud account using STS credentials.
...
```
Comment (addetz) [inline docs/docs-content/clusters/public-cloud/aws/add-aws-accounts.md:168]:
```suggestion
- If you are using a self-hosted instance of Palette or VerteX, you must configure an AWS account at the instance-level
```

File: docs/docs-content/vertex/system-management/configure-aws-sts-account.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["palette", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+import StsConfig from "../../../../_partials/self-hosted/_aws-sts-config.mdx";
+
+<StsConfig name="sts-config" edition="VerteX" />
```
Comment (addetz) [inline docs/docs-content/vertex/system-management/configure-aws-sts-account.md:14]:
```suggestion
<PartialsComponent category="self-hosted" name="sts-config" edition="VerteX" />
```

Comment (addetz) [reply docs/docs-content/vertex/system-management/configure-aws-sts-account.md:14]: Use your pal, the `PartialsComponent`

File: docs/docs-content/vertex/system-management/configure-aws-sts-account.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+title: "Enable Adding AWS Accounts Using STS "
+description: "This page teaches you how to allow tenants to add AWS accounts using STS."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["palette", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+import StsConfig from "../../../../_partials/self-hosted/_aws-sts-config.mdx";
```
Comment (addetz) [inline docs/docs-content/vertex/system-management/configure-aws-sts-account.md:12]:
```suggestion
```

File: docs/docs-content/enterprise-version/system-management/configure-aws-sts-account.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+title: "Enable Adding AWS Accounts Using STS "
+description: "This page teaches you how to allow tenants to add AWS accounts using STS."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["palette", "management", "account", "credentials"]
+keywords: ["self-hosted", "palette"]
+---
+
+import StsConfig from "../../../../_partials/self-hosted/_aws-sts-config.mdx";
```
Comment (addetz) [inline docs/docs-content/enterprise-version/system-management/configure-aws-sts-account.md:12]:
```suggestion
```

File: docs/docs-content/enterprise-version/system-management/configure-aws-sts-account.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+<StsConfig name="sts-config" edition="Palette" />
```
Comment (addetz) [inline docs/docs-content/enterprise-version/system-management/configure-aws-sts-account.md:14]:
```suggestion
<PartialsComponent category="self-hosted" name="sts-config" edition="Palette" />

```

**Review by addetz — APPROVED** [review]

**Review by tiwarisumit3 — APPROVED** [review]

File: _partials/self-hosted/_aws-sts-config.mdx (hunk 2 above)
Comment (lennessyy) [reply _partials/self-hosted/_aws-sts-config.mdx:49]: I tried to add indentation, but they actually mess up the command so I think I will just leave it this way

**Pull request merged**